src/haddock/gear/pretty_table.py

```diff
@@ -0,0 +1,264 @@
+"""
+Tools to help the different modules creating pretty tables.
+
+Given a table in dictionary format:
+
+.. code:: python
+
+        {
+            "col_name_1": [value1, value3],
+            "col_name_2": [value2, value4],
+            "col_name_3": [value5, value6],
+            }
+
+Creates the following table in text::
+
+    col_name_1    col_name_2    col_name_3
+      value1        value2        value5
+      value3        value4        valut6
+
+
+These tables are space-separated values, where the number of spaces
+varies to make proper alignment of the text. If you are using Python,
+you can parse these tables with, for example:
```

Comment:
```suggestion
These tables are space-separated values, where the number of spaces
varies to make proper alignment of the text, for purely aesthetic reasons. When parsing these tables one cannot rely on them following a regular spacing interval; the space between one column and the next might have 4 spaces, but then the next column may have 6, etc. Thus if these files are to be used for data analysis, the spacing must first be normalized. This can easily be done in Python, as seen in the example below. In other languages you might need to write a regular expression to achieve the same.
```

Reply:
I didn't use this text exactly but I updated the documentation. I think it is clearer and clear now. Let me know otherwise.